### PR TITLE
Added DevOps Skills

### PR DESCRIPTION
**Pull request by MuhBayu**

### CODE REVIEW

**Comment by g-savitha** [comment]
Hi @MuhBayu  please mention the issue to which you are commiting to. If issue is not created please create an issue and get it assigned to you to avoid redundancy of PRs :)

File: src/constants/skills.js [hunk 1]
```diff
@@ -115,6 +116,7 @@ const categorizedSkills = {
       "bash",
       "azure",
       "vagrant",
+      "circleci"
```
Comment (g-savitha) [inline src/constants/skills.js:119]: add quasar under devops section

**Review by g-savitha — CHANGES_REQUESTED** [review]
Good work @MuhBayu . Just a minor change is reqd. I have mentioned an inline comment in the commit

File: src/constants/skills.js (hunk 1 above)
Comment (MuhBayu) [reply src/constants/skills.js:119]: > add quasar under devops section

sorry sir, quasar is a framework right?

Comment (rahuldkjain) [reply src/constants/skills.js:119]: @g-savitha `quasar` is a framework.

**Review by rahuldkjain — APPROVED** [review]

File: src/constants/skills.js (hunk 1 above)
Comment (g-savitha) [reply src/constants/skills.js:119]: ok cool then :)

**Review by g-savitha — APPROVED** [review]

**Comment by g-savitha** [comment]
fixes https://github.com/rahuldkjain/github-profile-readme-generator/pull/152

**Pull request merged**